data/json/mapgen/nested/lodge_nested.json

```diff
@@ -17,9 +17,6 @@
         "               ",
         "               ",
         "               ",
-        "DF ARF LRL ORa ",
-        "R              ",
-        "SF DRA LRa ORO ",
         "DF ARF LRL LRa ",
         "R              ",
         "SF DRA LRa LRL ",
```

Comment:
Why not delete lines with just spaces? This isn't an exact repeat.

Reply:
Looking at the map this went with, there's only space for one of these sets of three lines in it; I'm pretty sure the others were added accidentally.